geode-core/src/main/java/org/apache/geode/redis/internal/RegionProvider.java

```diff
@@ -399,10 +399,10 @@ private void doInitializeList(ByteArrayWrapper key, Region r) {
     if (r != null)
       return r;
     do {
-      Result result = createRegionCmd.createRegion(key, defaultRegionType, null, null, true, true,
-          null, null, null, null, null, null, null, null, false, false, true, false, false, false,
-          true, null, null, null, null, null, null, null, null, null, null, null, null, null, false,
-          null, null, null, null, null, null, null, null, null, null, null);
+      Result result = createRegionCmd.createRegion(key, defaultRegionType, null, null, true, null,
+          null, null, null, null, null, null, null, false, false, true, false, false, false, true,
+          null, null, null, null, null, null, null, null, null, null, null, null, null, false, null,
+          null, null, null, null, null, null, null, null, null, null);
```

Comment:
There's some dead code just below here that could do with purging, if you were so inclined.